vuepress/docs/next/tutorials/samples/install-standard-demo.md

```diff
@@ -40,40 +51,42 @@ ent bundler from-git -d -r https://github.com/entando-samples/standard-demo-cont
 
 2. Log into your App Builder instance.
 
-3. Select Repository from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot
-   below.
+3. Select `Repository` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
    ![Repository.png](./images/Repository.png)
 
-4. Select `Install` for each bundle, where order of installation is important. The `standard-demo` bundle will need to be installed last, as it relies on MFEs from the other bundles to set up each of the pages. The `sd-banking` and `sd-customer bundles` will take a few minutes, because they include microservices that require the initialization of containers. 
+4. Select `Install` for each bundle, where order of installation is important. The `standard-demo` bundle will need to be installed last, as it relies on MFEs from the other bundles to set up each of the pages. 
+   ![Installed.png](./images/Installed.png)
+
+Each installation can take several minutes, while the application downloads the Linux images for the microservices and installs the related assets. The `sd-banking` and `sd-customer bundles` include microservices that require the initialization of containers and will take longer to install.
```

Comment:
the comma here is misleading and unnecessary. It makes you expect a different outcome for 'while the application' as compared to 'each installation'.

Reply:
agreed! my paranoid enthusiasm for commas must be checked ;)